### ISSUE TITLE
Changes privacy.userContext.longPressBehavior in about:config to 2 on startup

### ISSUE THREAD

**Issue by zsmall09** [mrdokenny/Cookie-AutoDelete#530]
As the title says, Cookie Autodelete changes privacy.userContext.longPressBehavior to 2 from whatever it was set to.  In my case I set it to 1. It works until I restart firefox, then it sets it back to 2.

**Comment by kennethtran93** [comment]
~~Just making sure - If you disabled just this extension, changed the value, and restarted, you would get the value you set previously?~~

~~Also...Just wondering if you had the multi-account webextension installed or not?~~

See updated note below.

**Comment by kennethtran93** [comment]
Also, please follow the issue template including: 
### Describe the Bug/Feature





### Screenshot in which the bug can be seen



### Steps for anyone to reproduce the bug



### Your settings 



- OS/version: 
- Browser/version: 
- Cookie AutoDelete version:

**Comment by kennethtran93** [comment]
**UPDATE**:  This is **not** a bug specific to this webextension.  

I've tried checking the longPressBehaviour option on Firefox Developer Edition using their Multi-Account Containers webextension before and after enabling it, and it's also doing the same thing (disabled = 0, enabled = 2). 

If I changed it to 1 and restarted Firefox, it changed that option back to 2 again.  This was all done while CAD was disabled.

May need to file a Firefox bug report for this.
